go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -97,6 +99,7 @@ func registerPluginFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&mysqlQueryTimeout, "mysql_server_query_timeout", mysqlQueryTimeout, "mysql query timeout")
 	fs.BoolVar(&mysqlConnBufferPooling, "mysql-server-pool-conn-read-buffers", mysqlConnBufferPooling, "If set, the server will pool incoming connection read buffers")
 	fs.DurationVar(&mysqlKeepAlivePeriod, "mysql-server-keepalive-period", mysqlKeepAlivePeriod, "TCP period between keep-alives")
+	fs.DurationVar(&mysqlServerFlushDelay, "mysql_server_flush_delay", mysqlServerFlushDelay, "Delay after which buffered response will be flushed to the client.")
```

Comment:
I'd like to keep it this way. Dunno if we have a generic plan to do this? Since I wouldn't want to do it just for a one off flag then which only happens due to a refactor that for most end users is not really relevant anyway.

Reply:
Yeah let's not do a one-off flag rename/deprecation. We should tackle all of them together.